### PR TITLE
Quotes Section Added

### PR DESCRIPTION
**Pull request by Randula98**

### CODE REVIEW

**Comment by Randula98** [comment]
Screenshot 2023-03-06 235127
Screenshot 2023-03-06 235243

File: src/components/client/landing/quotesSection/QuotesSection.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import quotes from './quotes.json'
+
+function QuotesSection() {
+  return (
+    <div className="container-fluid row px-2 pt-4 pb-4 quotesDiv">
+      <h1 className="text-center">Quotes</h1>
+      <Swiper modules={[Navigation, Pagination]} navigation pagination={{ clickable: true }}>
+        {
+          quotes.map((quote) => (
```
Comment (senuravihanjayadeva) [inline src/components/client/landing/quotesSection/QuotesSection.jsx]: First check whether quotes is existing or not

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

File: src/components/client/landing/quotesSection/QuotesSection.jsx (hunk 1 above)
Comment (Randula98) [reply src/components/client/landing/quotesSection/QuotesSection.jsx]: on it!

Comment (Randula98) [reply src/components/client/landing/quotesSection/QuotesSection.jsx]: Screenshot 2023-03-07 232332

File: src/components/client/landing/quotesSection/QuotesSection.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import 'swiper/css/scrollbar'
+
+import quotes from './quotes.json'
+
+function QuotesSection() {
+
+  return (
+    <div className="container-fluid row px-2 pt-4 pb-4 quotesDiv">
+      <h1 className="text-center">Quotes</h1>
+      {quotes.length === 0 ? (
```
Comment (senuravihanjayadeva) [inline src/components/client/landing/quotesSection/QuotesSection.jsx]: no need of  === 0. Please use !quotes.length

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

**Comment by senuravihanjayadeva** [comment]
@Randula98 please resolve conflicts

**Pull request merged**